First proposal of an organization README.md (#13)
phlowers/.github: .github/profile/README.md

## Welcome to our power grid full of...

> <img src="phlowers_logotxt.png" width="200" height="200" style="display: block; margin: 0 auto"/>

Reviewer: ```suggestion
<img src="phlowers_logotxt.png" alt="phlowers logo" width="200" height="200" style="display: block; margin: 0 auto"/>
```

> <img src="phlowers_logotxt.png" width="200" height="200" style="display: block; margin: 0 auto"/>

Reviewer 2: I would add an alt attribute here
Also, it's hard to read the text "phlowers" when using Github with a dark theme because then the first three letters are displayed in dark blue against dark grey. Adding a white background would help make it more readable.

Reviewer: Good point for the logo. The next one is with white background to avoid contrast issues

🙋‍♀️ A short introduction - what is our organization all about?

Phlowers is an open-source library suite for studying power lines, written in Python.

The suite includes:

> - **mecaphlowers**: a package dedicated to mechanical and geometrical calculations for overhead power lines.

Reviewer: ```suggestion
- **mechaphlowers**: a package dedicated to mechanical and geometrical calculations for overhead lines.
```

> - **thermohl**: a package focused on thermal analysis of power lines.

Reviewer: ```suggestion
- **thermohl**: a package focused on thermal analysis of overhead lines.
```

Author: It seems that you have change somewhere else for "overhead power lines", so we have to choose...
